### ISSUE TITLE
Column pg_database.datlastsysoid does not exist anymore since Postgresql v15

### ISSUE THREAD

**Issue by mgmgh** [schemaspy/schemaspy#1001]
## Expected Behavior
This generates an error but schemaspy can continue anyway...
I suppose the goal is to retrieve the database's comment in /src/main/resources/org/schemaspy/types/pgsql.properties with this query:
SELECT datname AS catalog_name, pg_catalog.obj_description(datlastsysoid, 'pg_database') AS catalog_comment FROM pg_catalog.pg_database WHERE datname = :catalog ;

## Context
* Database type (e.g. MySQL): Postgresql v15.2 on Debian stable

## Your Environment
* Version used: 6.1.1 snapshot
* Java version: openjdk 11 on Debian stable

**Comment by npetzall** [comment]
Unsure which databaseType you've used `-t`.
Could you have a try with `-t pgsql11` if that wasn't what was used.

**Comment by npetzall** [comment]
Hi, I'll try to recreate it and see what error is produced. It's the catalog comment query that might need an update for version 15.

**Comment by mgmgh** [comment]
It works (I can see database's comment in generated html !) if you replace the function name and the bad column with:
pg_catalog.**sh**obj_description(**oid**, 'pg_database')
(exists since pg v11+, don't know for older versions as the docs are no more online)

**Comment by npetzall** [comment]
@mgmgh since it works in version 11 and 15 would you like to contribute by making a pull request?

You should be able to go to https://github.com/schemaspy/schemaspy/blob/master/src/main/resources/org/schemaspy/types/pgsql11.properties

Choose edit (the pen)

Then just add your updated query
```
selectCatalogsSql=SELECT datname AS catalog_name, pg_catalog.shobj_description(oid, 'pg_database') AS catalog_comment FROM pg_catalog.pg_database WHERE datname = :catalog
```

**Commit changes**
**First field:**  `fix(pgsql11): update catalog query`

**Second field:**  `fixes #1001`

**Create new branch**
*Branch name, will appear when create new branch is selected:* `1001_update_catalog_query_pgsql11`
